released/packages/coq-mi-cho-coq/coq-mi-cho-coq.0.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "A specification of Michelson in Coq to prove properties about smart contracts in Tezos"
+maintainer: "raphael.cauderlier@nomadic-labs.com"
+authors: [ "Raphaël Cauderlier" "Bruno Bernardo" "Julien Tesson" "Arvid Jakobsson" ]
+
+homepage: "https://gitlab.com/nomadic-labs/mi-cho-coq/"
+dev-repo: "git+https://gitlab.com/nomadic-labs/mi-cho-coq/"
+bug-reports: "https://gitlab.com/nomadic-labs/mi-cho-coq/issues"
+license: "MIT"
+
+build: [
+  ["./configure"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  make "install"
+]
+depends: [
+  "coq" {>= "8.8"}
+  "coq-menhirlib" {>= "20190626"}
+  "coq-ott" {>= "0.29"}
+  "menhir"
+  "ocaml" {>= "4.07.1"}
+  "ocamlbuild"
+  "ott" {build & >= "0.29"}
+  "zarith"
+]
+
+description: """
+Michelson is a language for writing smart contracts on the Tezos blockchain.
+This package provides a Coq encoding of the syntax and semantics of Michelson,
+automatically generated by the Ott tool. Also included is a framework called Mi-Cho-Coq
+for reasoning about Michelson programs in Coq using a weakest precondition calculus."""
+tags: [
+  "category:Computer Science/Programming Languages/Formal Definitions and Theory"
+  "date:2021-06-15"
```

Comment:
Is that a typo? I would expect version 0.1 to have been released a few months ago already. To be clear, the date is meant to be the release date, not the packaging date. They often match, since developers tend to package their libraries the same day they release it. But here, it is a bit surprising to see that both versions 0.1 and 1.0.0 have the exact same release date.

Reply:
It is not a typo, I have created the two releases on Gitlab the same day.

Version 0.1 corresponds to the current master branch of Mi-Cho-Coq which has almost not moved for a year. Version 1 corresponds to the current dev branch on which most development happens. I fear that some users might use the master branch and upgrading to v1 might be hard for them I wanted to gave them a release corresponding to the version they are already using.

That being said, I am very new to Coq package releasing and don't have much experience with opam packaging in general so don't hesitate to tell me if what I do is weird.

Reply:
No, that is fine. I just wanted to make sure there was no misunderstanding on the meaning of the `date` tag.